hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieStorageType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2016 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.uber.hoodie.common.model;
+
+public enum HoodieStorageType {
```

Comment:
Should this be FileFormat type? This is more at the file level right?   

Today, we have table types (Copy_on_write, merge_on_read), which I am thinking about calling storage types going forward.. Then, a RO table and RT table will be table types supported for a storage type.. In general, we need to get this straight